larq_zoo/sota/quicknet.py

```diff
@@ -58,20 +57,35 @@ def squeeze_and_excite(inp: tf.Tensor, filters: int, r: int = 16):
     return tf.reshape(out, [-1, 1, 1, filters])
 
 
-@factory
-class QuickNetBaseFactory(ModelFactory):
-
+class QuickNetBaseFactory(ModelFactory, abc.ABC):
     name: str = "model"
-    blocks_per_section: Sequence[int] = Field(None)
-    section_filters: Sequence[int] = Field(None)
-    use_squeeze_and_excite_in_section: Sequence[bool] = Field(None)
-    transition_block: MethodType = Field(None)
+    blocks_per_section: Sequence[int] = Field()
+    section_filters: Sequence[int] = Field()
+    use_squeeze_and_excite_in_section: Sequence[bool] = Field()
+    transition_block: Callable[..., tf.Tensor] = Field()
```

Comment:
Here I've removed the `None` default values which are technically type errors (which don't ever throw because the values are always overriden).